templates/base.html

```diff
@@ -89,10 +93,6 @@
                                 <span class="icon-bar"></span>
                                 <span class="icon-bar"></span>
                             </button>
-                            <a class="navbar-brand hidden-xs" href="{% url 'home' %}">{% brand_name %}</a>
-                            <span class="navbar-brand visible-xs">
-                            {% brand_name %}{% if instance %} &nbsp; {{ course.name|parse_localization }}{% endif %}
-                            </span>
                         </div>
                         <div class="collapse navbar-collapse" id="bs-navbar-collapse">
                             <ul class="nav navbar-nav hidden-xs">
```

Comment:
I wonder if the course instance selection menu could be shown in the mobile view too. Could you try this?

Reply:
I think we can keep the `drop-down menu` in the `topbar`.

Reply:
What about this?

![image](https://user-images.githubusercontent.com/10437475/98563694-25e2a480-22b4-11eb-8032-57784fe819ff.png)


Or we can keep the dropdown menu next to the brand icon just as in the desktop version. 
![image](https://user-images.githubusercontent.com/10437475/98563542-f895f680-22b3-11eb-99c5-d2446a8e164d.png)
@Mankro which one do you like more?

Reply:
I like the second one (separate course instance menu and the toggle menu like in the desktop version).